airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_client.py

```diff
@@ -50,7 +50,7 @@ def mock_default_sleep_interval(mocker):
 @pytest.fixture(name="client")
 def client_fixture(some_config, requests_mock, fb_account_response):
     client = Client(**some_config)
-    requests_mock.register_uri("GET", FacebookSession.GRAPH + "/v10.0/me/adaccounts", [fb_account_response])
+    requests_mock.register_uri("GET", FacebookSession.GRAPH + "/v11.0/me/adaccounts", [fb_account_response])
```

Comment:
I think we should use `FacebookAdsApi.API_VERSION` this dependency is not good :/